audio/internal/convert/stereo.go

```diff
@@ -0,0 +1,116 @@
+// Copyright 2017 The Ebiten Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package convert
+
+import (
+	"fmt"
+	"io"
+)
+
+// Setero objects convert little-endian audio mono or stereo audio streams into
+// 16-bit stereo stereo streams.
+type Stereo struct {
+	source io.ReadSeeker
+	mono   bool
+	bytes  int
+	buf    []byte
+}
+
+// IsValidResolution returns true if the given bit resolution is supported.
+func IsValidResolution(r int) bool {
+	switch r {
+	case 8, 16, 32, 24:
```

Comment:
32bit is not the target. (I think 32bit can be float values https://www.sounddevices.com/32-bit-float-files-explained/)

Reply:
I see references to both PCM and float. It seems there's lots of abuse of the WAV file format. In any case, I'll eliminate it from this change.